src/scr.c

```diff
@@ -678,19 +679,37 @@ static int scr_get_params()
   /* if not enabled, bail with an error */
   if (! scr_enabled) {
     scr_param_finalize();
+    if (scr_my_rank_world == 0) {
+      // 1. Dump SCR parameters - SCR_ENABLE 
+      scr_dbg(0, "SCR_ENABLE is disabled");
+    }
```

Comment:
Let's actually not bother printing ``SCR_ENABLE`` right now.  That's a parameter that not really used.
```suggestion
```